models/valve_landmarks/configs/metadata.json

```diff
@@ -12,6 +13,7 @@
     "pytorch_version": "1.13.0",
     "numpy_version": "1.21.2",
     "optional_packages_version": {},
+    "name": "Valve landmarks regression",
```

Comment:
Hi @ericspod , the added "name" tag is to represent the display name of the bundle in the webpage. Could you please help to check the names? Feel free to make any modifications of all of your bundles. Thanks in advance!

Reply:
Hi @yiheng-wang-nv  I suggested using capital case for those I contributed but I didn't want to change other people's names.